R/with.R

```diff
@@ -44,12 +44,10 @@ modify_defaults <- function(defaults, ...) {
   to_null <- vapply(vals, is.null, logical(1L))
   if (!all(nms[to_null] %in% names(defaults))) {
     bad_nms <- setdiff(nms[to_null], names(defaults))
-    is_are <- if (length(bad_nms) > 1L) "are" else "is"
-    warning(
-      "Trying to remove ", glue_collapse(sQuote(bad_nms), sep = ", ", last = " and "),
-      ", which ", is_are, " not in `defaults`.",
-      call. = FALSE
-    )
+    is_are <- if (length(bad_nms) > 1L) "are" else "is" # nolint: object_usage_linter.
+    cli_warn(c(
+      i = "Trying to remove {.field {bad_nms}}, which {is_are} not in {.arg defaults}."
```

Comment:
We should use {cli} pluralization here, something like `{?is/are}`:

https://cli.r-lib.org/articles/pluralization.html

Reply:
Ah, we are using pluralization, but I had missed this instance somehow. Fixed it.